sql/src/test/java/io/crate/integrationtests/RestSQLActionIntegrationTest.java

```diff
@@ -68,6 +68,15 @@ public void testWithArgsAndBulkArgs() throws IOException {
         ));
     }
 
+    @Test
+    public void testEmptyBulkArgsWithStatementContainingParameters() throws IOException {
+        execute("create table doc.t (id int primary key) with (number_of_replicas = 0)");
+        CloseableHttpResponse response = post("{\"stmt\": \"delete from t where id = ?\", \"bulk_args\": []}");
+        assertThat(response.getStatusLine().getStatusCode(), is(200));
```

Comment:
Why does this return `200 OK`? The argument is not bound to any parameter.

Reply:
Because if the bulk_args are empty there is nothing to execute. This is basically a no-op. The statement can be parsed and analysed without bound params. So everything is fine.

I can update the test method name to make that more clear

Reply:
I would expect empty bulk args to throw an error if arguments are present. I see that you changed that with this PR. There is probably a good reason for changing it.

Reply:
> There is probably a good reason for changing it.

Yep to make it consistent with `stmt="delete from t", bulk_args=[]` which was also no-op. Empty bulk_args = nothing to execute. No matter if the statement contains parameters or not.

Reply:
Makes sense. Thanks for clarifying.